### PR TITLE
wpt: Enable tests in `/css/css-inline/baseline-source/`

### LINKED ISSUE #31875: Implement baseline-source
After #31705 I think it should be quite straightforward to implement `baseline-source`: https://drafts.csswg.org/css-inline/#baseline-source

This will require Stylo changes, so you can follow the instructions from #31723

Basically https://github.com/servo/stylo/blob/bcdebc41b2a2222e9fa5ae6d7d1572025c32dc47/style/properties/longhands/box.mako.rs#L107 needs to change the obsolete `servo-2013` into `servo`

And then https://github.com/servo/servo/blob/d814d0553981777c01409d545e5c1c335b8eb7f4/components/layout_2020/flow/inline.rs#L2145-L2153 needs to check `BaselineSource`, picking the 1st baseline for `BaselineSource::First`, the last baseline for `BaselineSource::Last`, and keep the current logic for `BaselineSource::Auto`.

### PR DESCRIPTION
**Pull request by MunishMummadi**
**Description**
-Tests for `baseline-source`

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes fix #31875 

addresses the  [issue](https://github.com/servo/servo/issues/31875)

### CODE REVIEW

File: Cargo.toml
Comment (Loirooriol) [inline Cargo.toml:1]: Don't modify this file

File: Cargo.lock
Comment (Loirooriol) [inline Cargo.lock:1]: Don't modify this file

File: tests/wpt/include.ini [hunk 1]
```diff
@@ -48,7 +48,7 @@ skip: true
   [css-highlight-api]
     skip: true
   [css-inline]
-    skip: true
```
Comment (Loirooriol) [inline tests/wpt/include.ini:51]: We should probably keep skipping the `initial-letter` and `text-box-trim` subdirectories.

So I would go with

```
  [css-inline]
    skip: true
    [baseline-source]
      skip: false
```

or

```
  [css-inline]
    skip: false
    [initial-letter]
      skip: true
    [text-box-trim]
      skip: true
```

There's also few tests in `animation`, `parsing`, and at the root. I don't think it matters much if they are enabled or not, but if enabled you will need to update the expectations.

**Review by Loirooriol — CHANGES_REQUESTED** [review]
Please change the title. This is not implementing baseline-source, it's just enabling tests.

File: tests/wpt/include.ini (hunk 1 above)
Comment (MunishMummadi) [reply tests/wpt/include.ini:51]: I will stick with the 1st option

**Review by Loirooriol — APPROVED** [review]
Looks good if https://github.com/servo/servo/actions/runs/8475945097 passes.

Sometimes tests behave differently in CI (e.g. because in your system you have different fonts), so you may need some manual adjustments if something fails.

**Comment by MunishMummadi** [comment]
Do u want me to update description. I will be on lookout for tests.

**Review by mrobinson — APPROVED** [review]

**Pull request merged**